.gitignore

```diff
@@ -118,8 +118,6 @@ venv.bak/
 .ropeproject
 
 # mkdocs documentation
-/site
```

Comment:
site was a default in the python gitignore. we're using it to handle all of the website content.